formatter.go

```diff
@@ -54,6 +55,11 @@ func (f *Formatter) Format(amount int64) string {
 	return sa
 }
 
+// ToMajorUnits returns float64 representing the value in sub units using the currency data
+func (f *Formatter) ToMajorUnits(amount int64) float64 {
```

Comment:
Hi,

It's really project specific feature. You can do it by extending money struct in your project.
Or just get amount by calling `Amount()` and do calculations later

Reply:
I disagree, a lot of other money libs in other languages have that [feature](https://github.com/RubyMoney/money/blob/master/lib/money/money.rb#L307) as its a fairly common use case.